swift/ql/lib/codeql/swift/generated/Locatable.qll

```diff
@@ -6,15 +6,25 @@ import codeql.swift.elements.Location
 
 module Generated {
   class Locatable extends Synth::TLocatable, Element {
+    /**
+     * Gets the location, if it exists.
```

Comment:
How easy is it to tweak these individually, e.g. if I want this to say `Gets the location associated with this element.` or `Gets the location of this element in the source.` or some such?  Which file would I need to edit in future if I wanted to accomplish this?

Reply:
Actually, you can achieve what you propose in the current version by editing `schema.py` like this
```diff
 @qltest.skip
 class Locatable(Element):
-    location: optional[Location] | cpp.skip
+    location: optional[Location] | cpp.skip | docname("location of this element in the code")
```
however that does not work very well currently with repeated properties, where one needs to switch between singular (normal getter) and plural (documenting `getA...` and `getNumberOf...`), because the generator would not necessarily know where to put the `s`.

Reply:
please look at the updated `location` doc now :slightly_smiling_face: